test/fixtures.coffee

```diff
@@ -874,22 +945,22 @@ sampleError = """
 """
 
 sampleHeaders = {
-"Content-Type": "application/json",
+"content-type": "application/json",
```

Comment:
This imho sucks. If you want to test some new case (introduce or verify some new funcionality), add fixtures and tests for that particular case and please do not change existing tests and fixtures .

I miss here some test like:
```
describe 'when expected and real headers keys are with different case', () ->
  it 'should not return any error', () ->
    ...
```